dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/taglibcacheconfig.jstl.2.0.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2021 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+
+<taglib-cache-config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="http://www.ibm.com/xml/ns/TagLibCacheConfig.xsd">
+	<implict-taglib>
+		<uri>http://websphere.ibm.com/tags/tsx</uri>
+		<prefix>tsx</prefix>
+		<location>/com/ibm/ws/jsp/tsx/tag/tsx.tld</location>
+	</implict-taglib>
+	<implict-taglib>
+		<uri>http://websphere.ibm.com/tags/jsx</uri>
+		<prefix>jsxFunction</prefix>
+		<location>/com/ibm/ws/jsp/jsx/tags/jsxFunction.tld</location>
+	</implict-taglib>
+	<global-taglib>
+		<jar-name>standard.jstl.2.0.jar</jar-name>
```

Comment:
Same as the original taglibcacheconfig.xml ([link](https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.jsp/resources/com/ibm/ws/jsp/resources/taglibcacheconfig.xml))  but contains a reference to the new jar on line 26.